src/components/EventCard/styles.scss

```diff
@@ -0,0 +1,36 @@
+.event-card {
```

Comment:
so we're actually going to use less instead of scss, one really good thing about less is that if you want to refer to .event-card title, instead of .event-card title { }

you can do

.event-card {
  font-family: Roboto;
  width: 267px;
  border-radius: 10px;

  title {
    font-weight: bold;
    font-size: 15px;
  }
}